lib/reek/source/source_locator.rb

```diff
@@ -8,64 +8,27 @@ module Source
     # Finds Ruby source files in a filesystem.
     #
     class SourceLocator
+      include Enumerable
+      extend Forwardable
+
+      def_delegator :sources, :each
```

Comment:
It's used implicitly down here via to_a: https://github.com/troessner/reek/pull/697/files#diff-7b9b794d247d6e79de6ee9446ea9a3feR31

Please also see: http://stackoverflow.com/questions/7384862/is-there-ruby-documentation-that-can-verify-that-enumerable-to-a-calls-each-inte